<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTriggerInfo</class>
 <widget class="QDialog" name="DialogTriggerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="labelXckLength">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;High time duration of XCK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;XCK length&lt;/span&gt; is the high time duration of the signal XCK. XCK signals the camera to start the readout and indicates the time, which is needed for one readout.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register XCKLEN and is measured once per measurement. The signal XCK can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>XCK length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxXckLength">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;High time duration of XCK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;XCK length&lt;/span&gt; is the high time duration of the signal XCK. XCK signals the camera to start the readout and indicates the time, which is needed for one readout.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register XCKLEN and is measured once per measurement. The signal XCK can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> μs</string>
       </property>
       <property name="maximum">
        <double>42949672.950000002980232</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxXckPeriod">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time duration between two positive edges of XCK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;XCK period&lt;/span&gt; is the time between two positive edges of XCK. This time between two readouts is the time when the sensor will collect charges, so this is the exposure time.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register XCKPERIOD and is measured once per measurement. The signal XCK can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> μs</string>
       </property>
       <property name="maximum">
        <double>42949672.950000002980232</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBonLength">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;High time duration of BON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;BON length&lt;/span&gt; is the high time duration of the signal BON (block on). BON signals the time which is needed for one block of readouts.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register BONLEN and is measured once per measurement. The signal BON can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> μs</string>
       </property>
       <property name="maximum">
        <double>42949672.950000002980232</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelBonPeriod">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time duration between two positive edges of BON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;BON period&lt;/span&gt; is the time between two positive edges of BON (block on).&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register BONPERIOD and is measured once per measurement. The signal BON can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>BON period</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBonPeriod">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time duration between two positive edges of BON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;BON period&lt;/span&gt; is the time between two positive edges of BON (block on).&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register BONPERIOD and is measured once per measurement. The signal BON can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> μs</string>
       </property>
       <property name="maximum">
        <double>42949672.950000002980232</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelBonLength">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;High time duration of BON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;BON length&lt;/span&gt; is the high time duration of the signal BON (block on). BON signals the time which is needed for one block of readouts.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register BONLEN and is measured once per measurement. The signal BON can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>BON length</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelXckPeriod">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time duration between two positive edges of XCK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;XCK period&lt;/span&gt; is the time between two positive edges of XCK. This time between two readouts is the time when the sensor will collect charges, so this is the exposure time.&lt;/p&gt;&lt;p&gt;The signal is measured by the PCIe card in a 10 ns resolution. The value is stored in the S0 register XCKPERIOD and is measured once per measurement. The signal XCK can also be set to the output O of the PCIe card by the setting TOR, which can be found under: Settings --&amp;gt; Edit --&amp;gt; Camera Settings --&amp;gt; Board X --&amp;gt; Other --&amp;gt; tor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>XCK period</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelBoard">
       <property name="text">
        <string>PCIe board</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBoxBoard">
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxBoard</tabstop>
  <tabstop>doubleSpinBoxXckLength</tabstop>
  <tabstop>doubleSpinBoxXckPeriod</tabstop>
  <tabstop>doubleSpinBoxBonLength</tabstop>
  <tabstop>doubleSpinBoxBonPeriod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTriggerInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTriggerInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
